src/Microsoft.DotNet.Cli.Utils/EmbedAppNameInHost.cs

```diff
@@ -13,14 +13,16 @@ public static class EmbedAppNameInHost
     {
         private static string _placeHolder = "c3ab8ff13720e8ad9047dd39466b3c8974e592c2fa383d4a3960714caef0c4f2"; //hash value embedded in default apphost executable
         private static byte[] _bytesToSearch = Encoding.UTF8.GetBytes(_placeHolder);
+
+        /// <param name="appBinaryFilePath">Full path to appBinary or relative path to appHostDestinationFilePath</param>
```

Comment:
If we think one parameter should be doc'd, let's doc the whole method.  It's a little out of place to have only one parameter doc'd.

Reply:
I agree.